src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -86,8 +127,49 @@
     }
 
     public RobotMap() {
+        Component.pdp = new PDP();
+
+        Component.leftWheelEncoder = new CANCoder(Port.CAN.leftWheelEncoder);
+        Component.rightWheelEncoder = new CANCoder(Port.CAN.rightWheelEncoder);
+        Component.canCoderConfiguration = new CANCoderConfiguration();
+        Component.leftWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.rightWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.leftWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+        Component.rightWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+
+        Component.leftWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.leftWheelAccelerationCap.enable();
+        Component.rightWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.rightWheelAccelerationCap.enable();
+
+        Component.leftDriveA = new Motor("leftDriveA", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.leftDriveA));
+        Component.leftDriveB = new Motor("rightDriveA", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.rightDriveA));
+        Component.rightDriveA = new Motor("leftDriveB", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.leftDriveB));
+        Component.rightDriveB = new Motor("rightDriveB", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.rightDriveB));
+
+        Component.controlPanel = new Motor("controlPanel", false, new CANTalonFX(Port.CANMotor.controlPanel)); // TODO:
```

Comment:
The control panel motor should be on Brake mode, or we should use PID... but I would highly recommend Brake mode.